kpops/components/streams_bootstrap/streams/streams_app.py

```diff
@@ -63,10 +65,25 @@ def helm_repo_config(self) -> HelmRepoConfig | None:
     def get_clean_up_helm_chart(self) -> str:
         return f"{self.config.streams_bootstrap_helm_config.repository_name}/{AppType.CLEANUP_STREAMS_APP.value}"
 
-    def destroy(self, dry_run: bool, clean: bool, delete_outputs: bool) -> None:
-        super().destroy(dry_run, clean, delete_outputs)
-        if clean:
-            self.clean(dry_run, delete_outputs, self.config.clean_streams_apps_schemas)
+    @override
+    def reset(self, dry_run: bool) -> None:
+        values = self.to_helm_values()
+        values["streams"]["deleteOutput"] = False
+        self._clean_app(
+            values=values,
+            dry_run=dry_run,
+            retain_clean_jobs=self.config.retain_clean_jobs,
+        )
+
+    @override
+    def clean(self, dry_run: bool) -> None:
+        values = self.to_helm_values()
+        values["streams"]["deleteOutput"] = True
+        self._clean_app(
+            values=values,
+            dry_run=dry_run,
+            retain_clean_jobs=self.config.retain_clean_jobs,
+        )
```

Comment:
these methods are very similar, perhaps you could extract some of it into `_clean_app`